examples/units_and_coordinates/north_offset_frame.py

```diff
@@ -0,0 +1,45 @@
+"""
+===============================================
+Creating an Offset frame using NorthOffsetFrame
+===============================================
+
+This example shows a possible use of `~sunpy.coordinates.NorthOffsetFrame`
+
+A common use for this is to create a frame derived from Heliographic, which
+has the north pole at some point of interest. In this new frame, lines of
+longitude form great circles radially away from the point, and lines of
+latitude measure angular distance from the point.
+In this example the new frame is shifted so the new north pole is at (20,
+20) in the Heliographic Stonyhurst frame. The new grid is overplotted in
+blue.
+
+"""
+
+##############################################################################
+# Import the required modules.
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.coordinates import NorthOffsetFrame
+from sunpy.data.sample import AIA_171_IMAGE
+
+##############################################################################
+# Now to create the offset frame.
+m = sunpy.map.Map(AIA_171_IMAGE)
+
+north = SkyCoord(20*u.deg, 20*u.deg, frame="heliographic_stonyhurst")
```

Comment:
```suggestion
north = SkyCoord(45*u.deg, 20*u.deg,
                 frame="heliographic_stonyhurst", obstime=aiamap.date)
```
To be safe, always specify `obstime`
For reasons

Reply:
Earlier you had mentioned the shift so that the new north pole is (20,20) but have used 45*u.deg here, can I get some insight on this?